components/Input.jsx

```diff
@@ -0,0 +1,35 @@
+import classNames from 'classnames'
+import React from 'react'
+
+export default function Input({label, type='text', id, value, placeholder, className, onChange, children}) {
+  onChange
+  return (
+    <>
+      <label 
+      className={classNames(
+        'mr-4 mb-4'
+      )}
+      >
+        <span>{label}</span>
+      
+        <input 
+        className={classNames(
+          'cursor-text',
+          'mt-1 pl-4 py-1 peer',
```

Comment:
¿Por qué agregaste la clase peer?

Reply:
Peer es necesario para cuando queremos mostrar alguna cosa abajo, entonces cuando lo lleguemos a utilizar tiene que tenerlo ya.

Por ejemplo en Tailwind "Styling based on sibling state [(peer-{modifier})"](https://tailwindcss.com/docs/hover-focus-and-other-states)

<img width="669" alt="Screen Shot 2022-04-15 at 8 53 19 AM" src="https://user-images.githubusercontent.com/28359931/163573041-34be7df1-9b04-4e00-b316-1d40ed6b1613.png">